packages/eslint-plugin/package.json

```diff
@@ -42,17 +42,16 @@
   },
   "dependencies": {
     "@typescript-eslint/experimental-utils": "2.18.0",
-    "eslint-utils": "^1.4.3",
+    "eslint-utils": "^2.0.0",
     "functional-red-black-tree": "^1.0.1",
     "regexpp": "^3.0.0",
     "tsutils": "^3.17.1"
   },
   "devDependencies": {
-    "@types/json-schema": "^7.0.3",
```

Comment:
nit

Reply:
we don't consume the types via the plugin - we consume them via `experimental-utils`.
https://github.com/typescript-eslint/typescript-eslint/blob/b24fbe8790b540998e4085174251fb4d61bf96b0/packages/eslint-plugin/src/rules/naming-convention.ts#L3

I did this on purpose to make sure that the versions were the same. IIRC I ran into a problem once when `experimental-utils` had one version and `eslint-plugin` had another, so there were two copies installed which caused weirdness.

Reply:
you are correct:

```
'json-schema' should be listed in the project's dependencies. Run 'npm i -S json-schema' to add it  import/no-extraneous-dependencies
```

Reply:
actually yeah! That was the entire reason for exposing it from `experimental-utils`.
You have to explicitly import the types, but it's a type-only module, so our lint rule failed.
By exporting it from the utils package, I could consolidate the lint disable into a single place.